CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+Changelog
+===
+
+### 3.12
+
+Release date: TBD, 2017
+
+* [JENKINS-40621](https://issues.jenkins-ci.org/browse/JENKINS-40621) - 
+Prevent leaked file descriptorswhen invoking `MavenEmbedderUtils#getMavenVersion()`
+([PR #5](https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/5))
+* Update to Guice `4.0-beta`
```

Comment:
Maven plugin depends on this lib, so there will be a need to update the dep there to fixJENKINS-40621